pom.xml

```diff
@@ -108,13 +108,13 @@
         <compiler.fork>false</compiler.fork>
 
         <!-- Spring-Boot target version -->
-        <spring-boot-version>2.6.6</spring-boot-version>
+        <spring-boot-version>2.6.8</spring-boot-version>
```

Comment:
Should we align to 2.7.0? Upstream did it last week, moreover https://github.com/jboss-fuse/camel-spring-boot/blob/camel-spring-boot-3.14.2-branch/archetypes/camel-archetype-spring-boot/pom.xml#L37 should be updated too